gff3tool/bin/gff3_QC.py

```diff
@@ -1,4 +1,4 @@
-#! /usr/local/bin/python2.7
+#! /usr/local/bin/python3.6
```

Comment:
What's the exact version we want to use ? 3.6 or 3.7 ?
However, I don't think either of them would be right. The exact 0.7 sub version was specified because we use some syntax only works for 2.7, but I don't remember that we use any special syntax after that. Therefore, to make the whole application with much more freedom, it would be great to just use version "3".

Reply:
Got it.